### PR TITLE
Symfony 4.3 depreciation on Treebuilder.

### PR DESCRIPTION
**Pull request by partikule**

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -4,17 +4,23 @@
...
 use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\HttpKernel\Kernel;
 
 class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
+        if (Kernel::VERSION_ID >= 40200) {
+            $treeBuilder = new TreeBuilder('octopush_sms');
```
Comment (umpirsky) [inline DependencyInjection/Configuration.php:14]: This works in the older versions of Symfony too, no need to if.

Comment (partikule) [reply DependencyInjection/Configuration.php:14]: What works in older version of Symfony ?  new TreeBuilder('octopush_sms') ?

Comment (umpirsky) [reply DependencyInjection/Configuration.php:14]: Yes.

Comment (partikule) [reply DependencyInjection/Configuration.php:14]: Ah.
Could you correct the lib to make it compliant to Symfony 4.3 ?

Comment (umpirsky) [reply DependencyInjection/Configuration.php:14]: Sorry, I don't maintain this bundle any more. But I think your fix will make it compatible.

Comment (partikule) [reply DependencyInjection/Configuration.php:14]: OK ! 
So will someone accept my pull request or make the needed modification ?

Comment (octopush) [reply DependencyInjection/Configuration.php:14]: Thanks for the PR !

**Pull request merged**